File: Python/Lista_02(Estrutura_Condicional)/Operadora.py
minutos = int(input("Digite a quantidade de minutos: "))

if minutos<=100:
    aPagar = 50.00
else:
    aPagar = 50.00 + 2.00*(minutos-100)

print(f"Valor a pagar: R$ {aPagar:.2f}")

File: C++/Lista_02(Estrutura_Condicional)/Operadora.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){

    int minutos;
    double valorAPagar;

    cout << "Digite a quantidade de minutos: ";
    cin >> minutos;

    if (minutos>100){
        valorAPagar = 50.00 + (minutos-100)*2.00;
    }
    else{
        valorAPagar = 50.00;
    }

    cout << "Valor a pagar: R$ " << fixed << setprecision(2) << valorAPagar;

    return 0;

}